pkg/daemon/ceph/osd/agent.go

```diff
@@ -565,6 +565,11 @@ func waitForRebalance(context *clusterd.Context, clusterName string, osdID int,
 					osdID, initialUsage, currUsage)
 			}
 
+			// when the initial OSD PG count is zero there is nothing to do here
+			if init.Pgs == "0" {
```

Comment:
what do you think about checking for `init.UsedKB == 0` here also?

Reply:
Should they be checked together (`&&`) or just one of them is enough for the check to pass (`||`)?

Reply:
i think the check should be `||`, meaning just one of them being 0 is good enough to bail out right away and not wait for any of the values to decrease.